pages/account/index.tsx

```diff
@@ -5,11 +5,24 @@ import React from "react";
 import { GetServerSideProps } from "next";
 
 import AccountFormContainer from "../../src/components/AccountFormContainer";
-import { homePageRedirect } from "../../src/utils/utils";
+import {
+  homePageRedirect,
+  redirectIfUserHasRegistered,
+} from "../../src/utils/utils";
+import { useRouter } from "next/router";
 
-function AccountPage(): React.ReactElement {
-  const { t } = useTranslation("common");
+interface AccountPageProps {
+  hasUserAlreadyRegistered?: boolean;
+}
 
+function AccountPage({
+  hasUserAlreadyRegistered,
+}: AccountPageProps): React.ReactElement {
+  const { t } = useTranslation("common");
+  const router = useRouter();
+  React.useEffect(() => {
```

Comment:
nitpick: use `useEffect`  as the exported function from "react" rather than `React.useEffect`. It's a small change but react is moving to this convention so the entire package is not imported into individual files (helps with tree shaking as well)

Reply:
The entire repo is import ing React due to an eslint default requiring it. I can remove it from the specific files I updated, but this might be work for a future refactor.

Reply:
Sounds good.